mta-archive-builder/run.sh

```diff
@@ -0,0 +1,15 @@
+#/bin/bash
```

Comment:
Who runs this ?
Is this part of the test and should it run during the build ?

Reply:
This is my test script basically.

In a ideal world the image should be produced (see `docker build -t mta .` below and the code afterwards should be triggered as some kind of a basic smoke test.

Reply:
sounds good. Maybe we should look into this https://docs.docker.com/docker-hub/builds/automated-testing/

Reply:
+1 for the `run.sh`